requirements.txt

```diff
@@ -1,13 +1,100 @@
+AccessControl==2.13.13
+Acquisition==2.13.8
+DateTime==2.12.7
+DocumentTemplate==2.13.2
+ExtensionClass==2.13.2
+Flask==0.10.1
+Flask-SQLAlchemy==2.0
+Jinja2==2.7.3
+MarkupSafe==0.23
+Missing==2.13.1
+MultiMapping==2.13.0
 PasteDeploy==1.5.2
+Persistence==2.13.2
+Products.BTreeFolder2==2.13.4
+Products.ExternalMethod==2.13.0
+Products.MIMETools==2.13.0
+Products.MailHost==2.13.1
+Products.OFSP==2.13.2
+Products.PythonScripts==2.13.2
+Products.StandardCacheManagers==2.13.0
+Products.ZCTextIndex==2.13.5
+Products.ZCatalog==2.13.27
+Record==2.13.0
+RestrictedPython==3.6.0
 SQLAlchemy==1.0.5
 WebOb==1.4.1
+WebTest==2.0.18
+Werkzeug==0.10.4
+ZConfig==2.9.1
+ZODB3==3.10.5
+Zope2==2.13.22
+ZopeUndo==2.12.0
 argparse==1.2.1
+beautifulsoup4==4.3.2
+cryptacular==1.4.1
+docutils==0.7
+initgroups==2.13.0
+itsdangerous==0.24
+mechanize==0.2.5
+pbkdf2==1.3
 psycopg2==2.6.1
+py==1.4.29
 pyramid==1.5.7
+pyramid-jinja2==2.5
+pyramid-tm==0.12
+pytest==2.7.2
+pytz==2013b
 repoze.lru==0.6
+six==1.9.0
+tempstorage==2.12.2
+transaction==1.1.1
 translationstring==1.3
 venusian==1.0
 waitress==0.8.9
 wsgiref==0.1.2
+zExceptions==2.13.0
+zLOG==2.11.1
+zc.lockfile==1.0.2
+zdaemon==2.0.7
+zope.annotation==3.5.0
+zope.broken==3.6.0
+zope.browser==1.3
+zope.browsermenu==3.9.1
+zope.browserpage==3.12.2
+zope.browserresource==3.10.3
+zope.component==3.9.5
+zope.configuration==3.7.4
+zope.container==3.11.2
+zope.contentprovider==3.7.2
+zope.contenttype==3.5.5
+zope.deferredimport==3.5.3
 zope.deprecation==4.1.2
+zope.dottedname==3.4.6
+zope.event==3.5.2
+zope.exceptions==3.6.2
+zope.filerepresentation==3.6.1
+zope.i18n==3.7.4
+zope.i18nmessageid==3.5.3
 zope.interface==4.1.2
+zope.lifecycleevent==3.6.2
+zope.location==3.9.1
+zope.pagetemplate==3.5.2
+zope.processlifetime==1.0
+zope.proxy==3.6.1
+zope.ptresource==3.9.0
+zope.publisher==3.12.6
+zope.schema==3.7.1
+zope.security==3.7.4
+zope.sendmail==3.7.5
+zope.sequencesort==3.4.0
+zope.site==3.9.2
+zope.size==3.4.1
+zope.sqlalchemy==0.7.6
+zope.structuredtext==3.5.1
+zope.tal==3.5.2
+zope.tales==3.5.3
+zope.testbrowser==3.11.1
+zope.testing==3.9.7
+zope.traversing==3.13.2
+zope.viewlet==3.7.2
```

Comment:
something odd has happened here.  You should not have all these `zope.*` packages in your requirements.txt file.  Do you have any idea how they got there?